### PR TITLE
fetching_data_from_backend

### PR DESCRIPTION
**Pull request by joshuaobinna**
work in progress .
trying to get user profile on signup

### CODE REVIEW

**Comment by enyeInc** [comment]
@joshuaobinna You have no made any progress on this ticket. Where is the work from this past week?

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+USER_EMAIL = smtp.gmail.com
+EMAIL_PASSWORD = agroeasy1#$##
+EMAIL_HOST = agroeasy2018@gmail.com
```
Comment (Simpcyclassy) [inline .env]: Is this meant to be pushed

File: src/components/signin/components/Signin.js [hunk 2]
```diff
@@ -50,16 +50,16 @@ class Signin extends React.Component {
...
+            signinStatus === SUCCESS ? 
+                message.success(`${siginData.user.firstName}  ${SUCCESS_MESSAGE}`, 5):          
                 message.error(siginData.title, 5);
         }
         resetSignState();
     }
-
-    render() {
+    
+    render() {
```
Comment (Simpcyclassy) [inline src/components/signin/components/Signin.js]: Please fix all the extra spaces

File: .env (hunk 1 above)
Comment (enyeInc) [reply .env]: no it is not

File: src/components/signin/components/Signin.js (hunk 2 above)
Comment (joshuaobinna) [reply src/components/signin/components/Signin.js]: Noted

File: .env (hunk 1 above)
Comment (enyeInc) [reply .env]: @joshuaobinna remove this files from your commits

**Pull request closed without merging**